requests-for-comments/2023-10-27-base3000-launch.md

```diff
@@ -0,0 +1,70 @@
+# Base 3000 & Notebooks launch
+
+We want to launch the ‘first version’ of PostHog 3000 after the next sprint, which means the week starting November 13th (+ 1 week if needed). We have decided to **include Notebooks** in this launch, so that we can have a more impactful launch that combines functionality and design changes.
+
+## Scope for launch
+
+- Dark Mode
+- Base 3000 [Check this mega issue for details](https://github.com/PostHog/posthog/issues/18089)
+- Notebooks side panel
+    
+<img width="674" alt="Screenshot 2023-10-27 at 10 38 50" src="https://github.com/PostHog/meta/assets/14750837/abb9e559-91ea-408b-a8fd-5105f3c4c626">
+
+## Out of scope for launch (amongst other things)
+
+- Sidebar (Index view)
+- Canvas-powered dashboards
+- Query builder
+- [Check this mega issue for details + the full list](https://github.com/PostHog/posthog/issues/18089)
+
+
+## Rollout plan
+- [ ] Base 3000 ready to release (10th Nov)
+- [ ] Internal and a/b testing starts (13th Nov)
+- [ ] Docs revamp and updates ready (24th Nov)
+- [ ] Product pages for 3000 / Notebooks (24th Nov)
+- [ ] Release [PostHog 3000 (Base 3000 scope)](https://app.posthog.com/feature_flags/8472) (27th Nov)
+- [ ] Release [Notebooks](https://app.posthog.com/feature_flags/8734) (4th Dec) (or together with Base 3000 if we feel docs are easier)
+
+
+## Pre-launch evaluation
+
+We decided that even the "base" version has enough major changes that we want to test that it both doesn't impact our key metrics negatively as well as (if possible) evaluate if it is overall an improvement, especially with new users.
+This might be hard to do or to get clear signals for whether it is an improvement (especially as it is only a step along the way to the end goal UI/UX).
+
+- [ ] **Signup/onboarding Experiment**: _just_ for the onboarding we could experiment with the styles separate from the rest of the app
+  - [ ] **Metrics**: Same as usual onboarding metrics?
+  - [ ] Important that we keep the style consistent from signup through to app usage as otherwise it will be a janky experience
+  - [ ] We mostly care about **negatively** impacting the flow
```

Comment:
Chatted with Ben this morning, mostly relating to this part of the launch. Summary of what we discussed:

1. Signup should be considered and tested separately from onboarding
2. Signup/login tests can wait until later, those screens needs to feel _super_ cohesive with the rest of the app
4. The new product-specific onboarding (scenes /products and /onboarding) will be rolled out to everyone soon, so we can ignore /ingestion
5. The new onboarding will be trivial to 3000-ize, so we should just do that and test it with the rest of the app experience so that flow (onboarding -> full app) feels cohesive
    - I feel like cohesiveness here is more important than signup -> onboarding
6. All signup / onboarding / logged out scenes will only be light mode to reduce work required here
7. We should wait until we have button styles mostly locked in so we only have to test once 
    - This is up for debate depending on how long until button styles are ready to go. If it will be a significant delay we can move forward without, for the onboarding at least

How this impacts experimentation:
- If we are testing logged-in-states separately from logged-out-states, we don't need to worry about persistence. 
- If we want we can test with new users only (org created after x date will capture most of these, though some people do have multiple orgs) 
- We want to make sure new people get a consistent experience from the moment they log in, even if we decide to make changes to 3000. 
  - We can either add a user/org property once someone logs in, or manually add that person (and everyone in their org?) to the feature preview
  - If a user/org was originally on 3000, they should always be left on 3000, even if we end the experiment. Consistency is more important at this stage, they will get improvements as we make them to 3000.